Documentation/configuration.md

```diff
@@ -133,7 +133,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 The security flags help to [build a secure etcd cluster][security].
 
 ##### -ca-file [DEPRECATED]
-+ Path to the client server TLS CA file.
++ Path to the client server TLS CA file. The recommended equivalent alternative is "--trusted-ca-file '' --client-cert-auth".
```

Comment:
Can we put all of these in backticks? What's up with the empty quotes? Do we need to explain that? I don't follow what this is doing.